sqlglot/lineage.py

```diff
@@ -110,18 +108,17 @@ def to_node(
         for c in set(select.find_all(exp.Column)):
             table = c.table
             source = scope.sources[table]
+            if source.expression and sources:
+                expression = exp.expand(source.expression, parsed_sources)
```

Comment:
why is this needed? why don't you do this in the pre?

Reply:
Okay, I'll update expand.